CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```

Comment:
Any specific reasons to require 3.18 ? (it is not default in Ubuntu/Debian LTS)

Reply:
Not much reasons.
[For PyTorch, it does require 3.10](https://github.com/pytorch/pytorch/blob/master/CMakeLists.txt#L1), while having a series of logic to deal with different possible versions. We may set it to 3.13, the minimum version that allows ninja.

Reply:
I think 3.13 should be fine as I have seen 3.16 as default versions (AFAIR).

Reply:
Updated to 3.13.